src/api/json/catalog.json

```
@@ -3549,6 +3549,12 @@
       "description": "YAML schema for a webman package",
       "fileMatch": ["*.webman-pkg.yml"],
       "url": "https://raw.githubusercontent.com/candrewlee14/webman/main/schema/pkg_schema.json"
+    },
+    {
+      "name": "Configuration",
+      "description": "A configuration file for hint",
+      "fileMatch": ["**/config-schema.json"],
```

@rayazanderson I think this is to match the user's config file name, so it should probably match `.hintrc` instead.
```suggestion
      "fileMatch": ["*.hintrc"],
```